src/main/java/ch/njol/skript/effects/EffChange.java

```diff
@@ -272,9 +285,9 @@ else if (mode == ChangeMode.SET)
 	protected void execute(Event e) {
 		Expression<?> changer = this.changer;
 		Object[] delta = changer == null ? null : changer.getArray(e);
+		List<ChangeMode> modes = List.of(ChangeMode.DELETE, ChangeMode.RESET, ChangeMode.TOGGLE);
```

Comment:
I'd rather see this list not being created here (e.g. a field), or, even better, stick to the original pattern (`mode != ChangeMode.DELETE && mode != ChangeMode.RESET && mode != ChangeMode.TOGGLE`)

Reply:
I tried to make the line readable. If I still use the old pattern, the line becomes too long, and unreadable. With that, it doesn't respect Skript's guidelines.

> At most 120 characters per line

That's why I prefered to use a list.